modules/nf-core/coreograph/main.nf

```diff
@@ -0,0 +1,38 @@
+process COREOGRAPH {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "labsyspharm/unetcoreograph:2.2.9"
+
+    input:
+    tuple val(meta), path(image)
+
+    output:
+    tuple val(meta), path("*[0-9]*.tif"), emit: cores
+    tuple val(meta), path("./masks/"), emit: masks
+    tuple val(meta), path("TMA_MAP.tif"), emit: TMA_map
+    tuple val(meta), path("centroidsY-X.txt"), emit: centroids
+
+    path "versions.yml" , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def VERSION = '2.2.9'
+
+    """
+    python /app/UNetCoreograph.py \\
+        --imagePath ${image}\\
+        --outputPath .\\
+        $args
+
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        coreograph: $VERSION
+    END_VERSIONS
+    """
+}
```

Comment:
Can you add stub? :p

Reply:
What do you mean by a stub ?

Reply:
A stub can be used to run the pipeline without actually executing the tools in the processes (useful to test the pipeline logic itself very quickly). You need to add a stub section for this to actually work (see [here](https://github.com/nf-core/modules/blob/0591cad3d725d5c21337f72e638507abf709f75e/modules/nf-core/bcftools/merge/main.nf#L47-L61) for an example). It basically mimics the execution of the module, but creates empty files instead